### ISSUE TITLE
Checking if account verification email has clickable URL

### ISSUE THREAD

**Issue by jivanyatra** [HHMI-HQ/BI-assessment-builder#114]
> Originally opened by @yannisbarlas on 2026-02-20 at 13:25:48 UTC

> **Migrated from GitLab**
> **Original Author:** Christopher Vargas (`@vargasc`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/69

The hyperlink in the email provided for account verification wasn’t recognized as a hyperlink to open a browser/new webpage by alt nav technology (dragon naturally speaking) and Voiceover

I believe this may have been corrected in the current version of the account verification email, but I tried to look back through tickets and did not see anything specifically covering this.

**Comment by jivanyatra** [comment]
> Originally commented by @yannisbarlas on 2026-02-20 at 16:47:26 UTC


> **Migrated Comment**
> **Author:** Dione Mentis (`@DioneMentis`)
> **Date:** 2023-03-30T06:15:27.711Z

`@conneelyb`  `@aleezaoshry`  -- We didn't get a confrimation for Chris before he left, but I believe this issue is resolved. Our team cannot replicate it. Can you confirm please

**Comment by jivanyatra** [comment]
> Originally commented by @yannisbarlas on 2026-02-20 at 16:47:28 UTC


> **Migrated Comment**
> **Author:** aleezaoshry (`@aleezaoshry`)
> **Date:** 2023-03-31T14:46:09.198Z

Hi `@DioneMentis`  - 
It's likely that this issue was related to what `@yannis`  mentions above about the mail clients issue (I remember we had a difficult time establishing logins for the testing protocols). We can run validation testing, but it would make the most sense to run validation testing for all of the issues identified during initial testing. Do you have a sense as to the status of resolving/remediating the identified issues?
